src/app/pages/OpenWalletPage/Features/FromPrivateKey/index.tsx

```diff
@@ -59,14 +61,28 @@ export function FromPrivateKey(props: Props) {
           htmlFor="privateKey"
           error={privateKeyIsValid === false ? t('openWallet.privateKey.error', 'Invalid private key') : ''}
         >
-          <Box border={false}>
-            <TextArea
+          <Box border={false} direction="row" round="small">
+            <TextInput
               id="privatekey"
               data-testid="privatekey"
+              role="privatekey-input"
```

Comment:
In the tests in 
`/src/app/pages/OpenWalletPage/Features/FromPrivateKey/__tests__/index.test.tsx` on L41, there's a TS error due to `textbox` being of type `HTMLElement` instead of a more specific type. I haven't added a type cast as I noticed there isn't one on develop either.